plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridPartition.java

```diff
@@ -0,0 +1,75 @@
+package org.jkiss.dbeaver.ext.cubrid.model;
+
+import org.jkiss.code.NotNull;
+import org.jkiss.code.Nullable;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
+import org.jkiss.dbeaver.model.impl.jdbc.JDBCUtils;
+import org.jkiss.dbeaver.model.meta.Property;
+import org.jkiss.dbeaver.model.struct.rdb.DBSTable;
+import org.jkiss.dbeaver.model.struct.rdb.DBSTablePartition;
+
+public class CubridPartition extends CubridTable implements DBSTablePartition{
+
+    private CubridTable parentTable;
+    private String expression;
+    private Integer[] expression_values;
+    private String comment;
+    public CubridPartition(
+            CubridTable table,
+            String name,
+            String type,
+            JDBCResultSet dbResult) {
+    super(table.getContainer(), name, type,dbResult);
+    this.parentTable = table;
+    
+   
+    expression = JDBCUtils.safeGetString(dbResult, "partition_expr");
+    expression_values = (Integer[]) JDBCUtils.safeGetObject(dbResult, "partition_values");
+    comment = JDBCUtils.safeGetString(dbResult, "comment");
+    this.setDescription(comment);
+  }
```

Comment:
Formatting is broken here
```suggestion
    public CubridPartition(
        @NotNull CubridTable table,
        @NotNull String name,
        @NotNull String type,
        @NotNull JDBCResultSet dbResult
    ) {
        super(table.getContainer(), name, type,dbResult);
        this.parentTable = table;
        expression = JDBCUtils.safeGetString(dbResult, "partition_expr");
        expression_values = (Integer[]) JDBCUtils.safeGetObject(dbResult, "partition_values");
        comment = JDBCUtils.safeGetString(dbResult, "comment");
        this.setDescription(comment);
    }
```